libraries/AP_GPS/AP_GPS.h

```diff
@@ -173,6 +173,7 @@ class AP_GPS
         bool have_gps_yaw_accuracy;       ///< does the GPS give a heading accuracy estimate? Set to true only once available
         uint32_t last_gps_time_ms;          ///< the system time we got the last GPS timestamp, milliseconds
         uint32_t uart_timestamp_ms;         ///< optional timestamp from set_uart_timestamp()
+        uint32_t lagged_sample_count;       ///< number of samples with 50ms more lag than expected
```

Comment:
Should be logged somewhere, particularly to use when people report the GPS is being unhealthy, or when trying to use this for diagnostics.